doc/man3/CONF_modules_load_file.pod

```diff
@@ -146,6 +154,25 @@ Load and parse configuration file manually, custom error handling:
      NCONF_free(cnf);
  }
 
+=head1 BUGS
+
+In OpenSSL 3.x the CONF_modules_load_file_ex() and CONF_modules_load_file()
+functions affect some shared global state other than the specified library
+context.
+Therefore, even if used with a freshly created library context, the function
+is not thread safe, and should not be used except as part of early application
+initialisation.
+Applications that want to use multiple library contexts may find that the most
+recently loadded configuration file perturbs prior settings in other library
+contexts.
+For example, B<ssl> module settings are not library context specific, and the
+last configuration file loaded that has an C<ssl_config> setting changes the
+SSL settings for all library contexts.
+The L<SSL_CTX_config(3)> and L<SSL_config(3)> functions can be used to perform
+late customisation of SSL contexts and connection handles after default
+configuration via the module's B<system_default> section, while using a single
+library context and configuration file.
```

Comment:
This sentence is too convoluted. Could you please split it and clarify? Especially the part `... after default
configuration via the module's B<system_default> section, while using a single library context and configuration file.`